src/serlo-editor-repo/plugin-text/index.tsx

```diff
@@ -35,6 +35,11 @@ const createTextPlugin = (
       return value
     },
     deserialize(value) {
+      // sometimes, the value is empty, e.g. https://github.com/serlo/frontend/issues/2411
```

Comment:
I'm not sure how helpful this comment is, as the production link from the issue will be working when this is deployed.

Reply:
true, will probably just remove it